### PR TITLE
feat: New openapi powered clients for interacting with integrations

### PR DESCRIPTION
**Pull request by tonyxiao**
And add a new openapi-fetch powered client

### CODE REVIEW

File: apps/api/index.ts [hunk 1]
```diff
@@ -15,6 +15,10 @@ app.set('trust proxy', true);
 const metricsApp = express();
 const port = process.env.SUPAGLUE_API_PORT ? parseInt(process.env.SUPAGLUE_API_PORT) : 8080;
 
+if (process.env.GLOBAL_AGENT_HTTP_PROXY) {
```
Comment (asdfryan) [inline apps/api/index.ts:18]: Do we need to add this to the docker-compose?

**Review by asdfryan — COMMENTED** [review]
Does this maintain all of the existing error handling (e.g. autopauses etc)?

**Review by tomkit — APPROVED** [review]
minor comment: can we name generated files '.gen.' too for clarity?

**Comment by tonyxiao** [comment]
I'll add both of those in a follow up PR. 👍

File: apps/api/index.ts (hunk 1 above)
Comment (tonyxiao) [reply apps/api/index.ts:18]: Added it to docker-compose file already , for local debugging

**Pull request merged**